packages/react-native-reanimated/scripts/format-cmake.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if ! which cmake-format >/dev/null; then
+  echo "error: cmake-format is not installed"
+  exit 1
+fi
+
+cmake-format -i "$@"
```

Comment:
At the moment we use the defaults for formatting. We could make some config if we want to, but I don't think we are opinionated about it.